### ISSUE TITLE
Add GPT Home Page to janus-idp.github.io

### ISSUE THREAD

**Issue by Zaperex** [janus-idp/janus-idp.github.io#153]
## Description
Provide a GPTs Home page to [the Janus community site](https://janus-idp.io/) to provide discovery of the details of a list of Sample GPTs which we have available.

## Acceptance Criteria
The homepage should include the following:
- **Header** containing:
  - `Sample GPTs`
  - `Have a GPT idea?`
  - Button -> `Submit a proposal for a GPT!`

The CTA should reference a new Issue template for [the software templates repo](https://github.com/janus-idp/software-templates/issues) similar to this [plugin issue template](https://github.com/janus-idp/backstage-plugins/issues/new?assignees=&labels=plugin&projects=&template=plugin.yaml&title=%F0%9F%94%8C+Plugin%3A+)

- **Search**
  - Include a search field, but do not provide the filter dropdown as the Plugins page has

- **GPT Cards**
  - Each card should contain:
    - Title
    - Description
    
 ## Notes
- GPTsList should include:
  - Title
  - Description
  - href
  - githubUrl

As a starter, include the following in the GPTs Home page:

https://github.com/janus-idp/software-templates/tree/main/scaffolder-templates/ansible-job
https://github.com/janus-idp/software-templates/tree/main/scaffolder-templates/quarkus-web-template
https://github.com/janus-idp/software-templates/tree/main/scaffolder-templates/documentation-template

## Blocker
The following issue(s) will need to be completed before this issue can be started:
- [x] https://github.com/janus-idp/software-templates/issues/136

**Comment by Zaperex** [comment]
UPDATE: The Golden Path Templates to add to the homepage should be updated to be the following:

- https://github.com/janus-idp/software-templates/blob/main/scaffolder-templates/go-backend-template/template.yaml
- https://github.com/janus-idp/software-templates/blob/main/scaffolder-templates/nodejs-backend-template/template.yaml
- https://github.com/janus-idp/software-templates/blob/main/scaffolder-templates/python-backend-template/template.yaml
- https://github.com/janus-idp/software-templates/blob/main/scaffolder-templates/dotnet-frontend-template/template.yaml
- https://github.com/janus-idp/software-templates/blob/main/scaffolder-templates/spring-boot-backend-template/template.yaml
- https://github.com/janus-idp/software-templates/blob/main/scaffolder-templates/quarkus-backend-template/template.yaml
- https://github.com/janus-idp/software-templates/blob/main/scaffolder-templates/argocd-template/template.yaml
- https://github.com/janus-idp/software-templates/blob/main/scaffolder-templates/ansible-job/template.yaml
